WalletWasabi.Fluent/Views/Dialogs/ShuttingdownDialogView.axaml

```diff
@@ -8,10 +8,14 @@
              x:DataType="dialog:ShuttingDownViewModel"
              x:CompileBindings="True"
              x:Class="WalletWasabi.Fluent.Views.Dialogs.ShuttingDownView">
-  <c:ContentArea Title="{Binding Title}"
-                 ScrollViewer.VerticalScrollBarVisibility="Disabled">
+  <c:ContentArea
+                 ScrollViewer.VerticalScrollBarVisibility="Disabled"
+                 NextContent="Ok" EnableNext="True">
+    <c:ContentArea.Title>
+      <TextBlock TextAlignment="Center" Text="{Binding Title}" />
+    </c:ContentArea.Title>
     <DockPanel VerticalAlignment="Center">
-      <TextBlock Text="This may take a few minutes."
+      <TextBlock Text="Wasabi is unable to close right now. Coinjoin in progress."
```

Comment:
I have concerns with the message. 
The previous one said "few minutes" but this one doesn't say how long it takes. I am afraid if we don't say some users will force close the software because they don't know how much to wait.

Reply:
@soosr Latest commit has a more clear message.